sensiolabs/gotenberg-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+    "bundles": {
+        "Sensiolabs\\GotenbergBundle\\SensiolabsGotenbergBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "GOTENBERG_DSN": "http://localhost:3000"
+    },
+    "docker-compose": {
+        "compose.yaml": {
+            "services": [
+                "gotenberg:",
+                "  image: 'gotenberg/gotenberg:8'"
+            ]
+        },
+        "compose.override.yaml": {
+            "services": [
+                "gotenberg:",
+                "  ports:",
+                "    - \"3000:3000\""
```

Comment:
In other services, we don't force the local port so that you can work on several projects with the same service in // without conflict.

Reply:
The override file is serving that purpose am I wrong ? It is not required though to expose. Depends if developers are using full docker network or the Symfony cli. Let me know if you want us to remove it (on mobile ATM so can't check for sure but I think I picked this idea from another recipe...)

Reply:
Fixed according to other examples in recipes.